backend/cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -110,9 +120,9 @@ function hackerSortingAlgo () {
   } else {
     matchAnswers()
     printMembers()
-    writeDataToFile()
+    writeDataToFile()    
   }
 }
 
 // let's get sorting!!
-hackerSortingAlgo()
```

Comment:
don't call this directly in the file, call within your main sorting process. similar callout to https://github.com/HackBeanpot/internal-tools/pull/137/files#diff-2b2de8166d35c5f271eae616c700b6841e5920da821789a78a1ac62963e760f3

Reply:
The main sorting process is done in the hackerSortingAlgo.ts file, there is no main file that calls the script like in judging algo's index.ts for the parser function. For this one I would have to leave this function here I believe